src/daikinapi.ts

```diff
@@ -300,7 +308,8 @@ export class DaikinApi{
       let requestedData = {};
       let autoHsp = deviceData.hspHome;
       switch(deviceData.mode){
-        case 1: //heat
+        case TargetHeatingCoolingState.HEAT:
+        case TargetHeatingCoolingState.AUXILIARY_HEAT:
```

Comment:
Not sure if this differs between Daikin setups but my Daikin One system with heat pump and gas furnace allows me to set a target temperature in both heating modes.